CHANGES/479.feature

```diff
@@ -0,0 +1,5 @@
+Updated the image building feature:
+    * Replaced `artifacts` by `build_context` (i.e., a file plugin repository version HREF)
+    as the parameter to provide the build context for a Containerfile.
+    * Added the `containerfile_name` as the parameter to provide the relative-path of a File Content
+    from the provided `build_context`.
```

Comment:
Reminder to all of us:
Towncrier does not do Markdown validation. So we need to be on the lookout if this produces a nice and readable changelog in the end.